django/contrib/admin/static/admin/css/responsive.css

```diff
@@ -0,0 +1,984 @@
+/* Tablets */
+
+@media (max-width: 991px) {
+    html, body {
+        height: 100%;
+    }
+
+    #container {
+        min-width: 0;
+    }
+
+    .dashboard #content {
+        width: auto;
+    }
+
+    .inline-group {
+        overflow: auto;
+    }
+
+    input[type="submit"], button {
+        -webkit-appearance: none;
+        appearance: none;
+    }
+
+    .aligned .form-row {
+        max-width: calc(100vw - 100px);
+        overflow: auto;
+    }
+
+    .selector {
+        display: flex;
+        width: 100%;
+    }
+
+    .selector select {
+        width: 100%;
+    }
+
+    .selector .selector-filter {
+        display: flex;
+        align-items: center;
+    }
+
+    .selector .selector-filter label {
+        margin: 0;
+    }
+
+    .selector-available, .selector-chosen {
+        width: auto;
+        flex: 1 1;
+    }
+
+    .selector ul.selector-chooser {
+        align-self: flex-start;
+    }
+
+    .selector .selector-available input {
+        width: auto;
+        flex: 1 1;
+    }
+
+    .stacked {
+        flex-wrap: wrap;
+        max-width: 490px;
+    }
+
+    .stacked > * {
+        flex: 0 0 100%;
+    }
+
+    .stacked .selector-chooser {
+        flex-basis: 50px;
+        margin-left: auto !important;
```

Comment:
is !important required in light of passing efforts to remove it (https://code.djangoproject.com/ticket/11544)?